configureRelease

```
@@ -47,6 +49,20 @@ class ExceptionHandlingIterator(object):
         except Exception as e:
             return self.__next__()
 
+def compareVersion(versionA, versionB):
```

Do we really need a wrapper for this? Does the KeyError happen frequently? If yes, it will be confusing anyway. If no, I would rather let it fail hard.

---

In the beginning it was failing quite often due to a typo. It the end it was not failing anymore. So yes I can remove the wrapper...